packages/components/src/components/as-category-widget/README.md

```diff
@@ -0,0 +1,190 @@
+Category widget displays the categories passed to the component as a list with a bar representing the percentage of each category in relation to the maximum category value.
+
+```html
+noSource: true
+---
+<iframe src="/examples/components/as-category-widget/simple.html" style="width: 100%; height: 330px;">
+```
+
+```code
+lang: html
+showSource: false
+---
+<as-category-widget
+  heading="Business Volume"
+  description="Description"></as-category-widget>
+
+<script>
+  const categoryWidget = document.querySelector('as-category-widget');
+  categoryWidget.showClearButton = true;
+  categoryWidget.categories = [
+    { name: 'Bars & Restaurants', value: 1000, color: '#FABADA' },
+    { name: 'Fashion', value: 900 },
+    { name: 'Grocery', value: 800 },
+    { name: 'Health', value: 400 },
+    { name: 'Shopping mall', value: 250 },
+    { name: 'Transportation', value: 1000 }
+    { name: 'Leisure', value: 760 }
+  ];
+</script>
+```
+
+### Props
+
+#### **categories** (array required)
```

Comment:
I think the most common way to document APIS in markdown is:

Required parameter:
name:type

Optional parameter:
[name:type]=default


cc @ivanmalagon @rubenmoya